browser/brave_drm_tab_helper.cc

```diff
@@ -92,14 +95,20 @@ void BraveDrmTabHelper::DidStartNavigation(
 
 void BraveDrmTabHelper::OnWidevineKeySystemAccessRequest() {
   is_widevine_requested_ = true;
+#if BUILDFLAG(IS_ANDROID)
+  bool for_restart = true;
```

Comment:
Just curious - android also needs restart for using drm?

Reply:
Actually just need the user to completely close the tab which requires Widevine (not refreshing) and re-open it, because the render client won't re-init with only a refresh. But that is too confusing and we decided o just relaunch the whole browser.